apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -228,7 +228,7 @@ const InstructionsPanel: React.FunctionComponent<InstructionsPanelProps> = ({
               id="instructions-feedback-message"
               className={moduleStyles['message-' + theme]}
             >
-              {offerBrowserTts && useMessage && (
+              {offerBrowserTts && useMessage && !canShowNextButton && (
```

Comment:
It could be worth a comment to indicate that navigation and TTS aren't inherently linked but that we avoid rendering it when there's a "next" button for UI weirdness, but either way is fine.